packages/vuetify/src/components/VCombobox/__tests__/VCombobox.spec.ts

```diff
@@ -319,4 +319,17 @@ describe('VCombobox.ts', () => {
 
     expect(wrapper.vm.$attrs.autocomplete).toBe('on')
   })
+
+  it('should pass the name attribute to the inner input element', async () => {
```

Comment:
The **genHiddenInput** method is never overwritten and this is already tested in `v-select` https://github.com/vuetifyjs/vuetify/pull/10327/files#diff-5b5fcac8ce361d1dfbffa8cdea0bca32R453.